docs/uninstall-olm.md

```diff
@@ -0,0 +1,46 @@
+# How do I uninstall OLM?
+
+OLM is installed within a namespace called `openshift-operator-lifecycle-manager` on an OpenShift cluster. On a local minikube cluster, it is installed within the namespace `olm`. You can uninstall OLM in a symmetrical way as you did to install it. 
+
+## Uninstall Released OLM
+For uninstalling release versions of OLM, for example version 0.13.0, you can use the following command:
+
+```bash
+export olm_release=0.13.0
+kubectl delete -f https://github.com/operator-framework/operator-lifecycle-manager/releases/download/${olm_release}/crds.yaml
+kubectl delete ns <OLM namespace, e.g.: olm, openshift-operator-lifecycle-managment, etc>
+```
+
+To find out the version of the OLM you are using, please use the following to see the version of packageserver csv.
+
+```bash
+oc -n <OLM namespace, e.g.: olm, openshift-operator-lifecycle-management, etc> get csvs
+NAME DISPLAY VERSION REPLACES PHASE
+packageserver Package Server 0.13.0 Succeeded
+```
+
+## Uninstall From Git Repository Master Branch
+
+You can also uninstall OLM from the master branch of the [operator-framework/operator-lifecycle-manager](https://github.com/operator-framework/operator-lifecycle-manager/) repository with the following: 
+
+```bash
+kubectl delete -f https://raw.githubusercontent.com/operator-framework/operator-lifecycle-manager/master/deploy/upstream/quickstart/crds.yaml
+kubectl delete ns <OLM namespace, e.g.: olm, openshift-operator-lifecycle-managment, etc>
+```
+If you have cloned git repository and use the [Makefile](https://github.com/operator-framework/operator-lifecycle-manager/blob/master/Makefile) for deploying OLM locally on [minikube](https://kubernetes.io/docs/tasks/tools/install-minikube/) for development purposes, you can remove OLM using the following commands:
+
+```bash
+git clone https://github.com/operator-framework/operator-lifecycle-manager.git
+cd operator-lifecycle-manager
+kubectl delete -f ./deploy/upstream/quickstart/crds.yaml
+kubectl delete ns operator olm
+```
+
+## Verify OLM Uninstall
+
+You can verify the necessary CustomResourceDefinitions are removed from applying the `crds.yaml` file with the following:
```

Comment:
This sentence might need restructuring. 
Maybe: "You can verify that olm has been uninstalled successfully by making sure all the relevant crds have been delete".
We should also probably check to see that the olm-operator and catalog-operator pods have also been deleted.

Reply:
yeah, the thing is, the olm ns should also be gone ... I'll try make it in there.